node/clean-up-filesystem.sh

```diff
@@ -315,9 +321,11 @@ packages_to_remove=$(microdnf repoquery --installed |
 
 
 echo "Removing ${packages_to_remove}"
+
 # Removing one of the packages deletes rc.local, move it out of the way.
 mv /etc/rc.local /etc/rc.local.bak
-rpm -e --nodeps $packages_to_remove
+# rpm -e --nodeps $packages_to_remove
```

Comment:
For now this is a hack because I was struggling to get both iptables and nftables in the same image, and I didn't want to spend too much time on it. Need to revisit.